packages/core/tests/integration/components/navi-visualizations/pie-chart-test.ts

```diff
@@ -277,27 +270,22 @@ module('Integration | Component | pie chart', function(hooks) {
      * Resize the parent element of the SVG that the pie chart is drawn in
      * This effectively moves the pie chart to the left
      */
-    find('.pie-chart-widget').style.maxWidth = '1000px';
+    (find('.pie-chart-widget') as Element & { style: { maxWidth: string } }).style.maxWidth = '1000px';
```

Comment:
Maybe this will work?
```suggestion
    (find('.pie-chart-widget') as HTMLElement).style.maxWidth = '1000px';
```